test/load/terraform/consul.tf

```diff
@@ -37,9 +37,9 @@ module "consul_servers" {
 
   vpc_id                  = module.vpc.vpc_id
   subnet_ids              = module.vpc.public_subnets
-  allowed_ssh_cidr_blocks = ["0.0.0.0/0"]
+  allowed_ssh_cidr_blocks = ["10.0.0.0/16"]
```

Comment:
Should all of these CIDRS come from `var.vpc_cidr`? That way if the VPC's network addressing is not 10.0.0.0/16 we can still get to them from within the VPC?